# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# Thomas Huet, 2022
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-12-03 14:33-0800\n"
"PO-Revision-Date: 2021-12-03 22:37+0000\n"
"Last-Translator: Thomas Huet, 2022\n"
"Language-Team: French (https://www.transifex.com/archesproject/teams/56656/fr/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: fr\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: arches/app/datatypes/base.py:38
#, python-brace-format
msgid "{0} error, {1} {2} - {3}. Unable to save."
msgstr ""

#: arches/app/datatypes/base.py:204
msgid "Multiple provisional edits. Returning first edit"
msgstr ""

#: arches/app/datatypes/base.py:208
msgid "Tile has no authoritative or provisional data"
msgstr ""

#: arches/app/datatypes/concept_types.py:111
msgid ""
"The widget used to save this data appears to be incorrect for this datatype."
" Contact system admin to resolve"
msgstr ""

#: arches/app/datatypes/concept_types.py:119
msgid "This is an invalid concept prefLabel, or an incomplete UUID"
msgstr ""

#: arches/app/datatypes/concept_types.py:127
msgid "This UUID is not an available concept value"
msgstr ""

#: arches/app/datatypes/datatypes.py:98
msgid "This is not a string"
msgstr ""

#: arches/app/datatypes/datatypes.py:227
msgid "Not a properly formatted number"
msgstr ""

#: arches/app/datatypes/datatypes.py:311
msgid "Not of type boolean"
msgstr "N'est pas de type boolÃ©en"

#: arches/app/datatypes/datatypes.py:392
msgid ""
"Incorrect format. Confirm format is in settings.DATE_FORMATS or set the "
"format in settings.DATE_IMPORT_EXPORT_FORMAT."
msgstr ""

#: arches/app/datatypes/datatypes.py:452
#, python-brace-format
msgid "{value} is an invalid date format"
msgstr ""

#: arches/app/datatypes/datatypes.py:527
msgid ""
"Incorrect Extended Date Time Format. See "
"http://www.loc.gov/standards/datetime/ for supported formats."
msgstr ""

#: arches/app/datatypes/datatypes.py:580
msgid ""
"Only dates that specify an exact year, month, and day can be used with the "
"\"=\" operator"
msgstr ""

#: arches/app/datatypes/datatypes.py:589
msgid ""
"Only dates that specify an exact year, month,"
"                                     and day can be used with the \">\", "
"\"<\", \">=\", and \"<=\" operators"
msgstr ""

#: arches/app/datatypes/datatypes.py:601
msgid "Invalid date specified."
msgstr ""

#: arches/app/datatypes/datatypes.py:668
msgid "Unable to serialize some geometry features"
msgstr ""

#: arches/app/datatypes/datatypes.py:1263
#, python-brace-format
msgid "This node has a limit of {0} files. Please reduce files."
msgstr ""

#: arches/app/datatypes/datatypes.py:1271
#, python-brace-format
msgid ""
"This node has a file-size limit of {0}. Please reduce file size or contact "
"your sysadmin."
msgstr ""

#: arches/app/datatypes/datatypes.py:1278
#, python-brace-format
msgid "datatype: {0}, value: {1} - {2} ."
msgstr ""

#: arches/app/datatypes/datatypes.py:1336
msgid "File does not exist"
msgstr ""

#: arches/app/datatypes/datatypes.py:1440
msgid "The file url is invalid"
msgstr ""

#: arches/app/datatypes/datatypes.py:1442
msgid "A file is not available for this tile"
msgstr ""
